src/components/pages/muster-page/muster-page.tsx

```diff
@@ -41,11 +41,12 @@ import {
   ApiRosterColumnType,
```

Comment:
There are a couple linting errors in this file.

Reply:
I don't get an error here. Can you let me know what you're seeing?

Reply:
![image](https://user-images.githubusercontent.com/581357/105503856-15c44980-5c95-11eb-9910-e8e826713eea.png)

Reply:
It looks like the linting errors were fixed. I'm not seeing any now.